samples/scenarios/jdbc_kerberos/README.md

```diff
@@ -0,0 +1,152 @@
+(source - [Oracle Java docs](https://docs.oracle.com/en/java/javase/11/security/java-generic-security-services-java-gss-api1.html)).  
+
+JAAS authentication is typically performed prior to secure communication using Java GSS-API. Tableau Server uses JAAS and JGSS for Server Runas and Kerberos Delegation authentication.
+
+## <a id="desktop"></a>Kerberos SSO for Tableau Desktop
+
+### SSPI authentication for Tableau Desktop on Windows
+connection-fields.xml
+```
+<field name="authentication" label="Authentication" category="authentication" value-type="selection">
+  <selection-group>
+    <option value="auth-integrated" label="Integrated" />
+    <!-- other auth options -->
+  </selection-group>
+</field>
+```
+connectionProperties.js
+```
+    props["gsslib"] = "gssapi";	 
+    props["jaasLogin"] = "false";  
+    props["jaasApplicationName"] = "com.sun.security.jgss.krb5.initiate";
+```
+
+![Image](images/DesktopConnectionDialog.png)
+
+### Kerberos SSO for Tableau Desktop on Mac
+connection-fields.xml
+```
+<field name="authentication" label="Authentication" category="authentication" value-type="selection">
+  <selection-group>
+    <option value="auth-integrated" label="Integrated" />
+    <!-- other auth options -->
+  </selection-group>
+</field>
+```
+
+connectionProperties.js
+```
+    props["jaasApplicationName"] = "com.sun.security.jgss.krb5.initiate";
+    props["gsslib"] = "gssapi";	 
+    props["jaasLogin"] = "false";  
+```
+
+Other Configuration:
+* Only for postgres jdbc plugin on Mac - add this property `Settings.DisableNativeGSS` as a `Boolean/YES` to the Tableau plist on Mac at `/Library/Preferences/com.tableau.Tableau-<version>.plist` as described below. Here version is Tableau Desktop major version, for example, `com.tableau.Tableau-2020.4.plist`.
+```
+// Note - the below command requires root privileges hence sudo is needed.
+sudo defaults write /Library/Preferences/com.tableau.Tableau-<version>.plist Settings.DisableNativeGSS -bool YES
+```
+* Make sure a krb5.conf file is present at `/etc/krb5.conf`(/etc is a private directory, requires root privileges). Check for the existence of either of the following two files `/etc/krb5.conf` or `/Library/Preferences/edu.mit.Kerberos`. If the second file (edu.mit.Kerberos) is present it needs to be backed up and deleted.
+* Before connecting, run `klist` to verify kerberos TGT for the user principal is present in the kerberos ticket cache. If not, run `kinit user@REALM` before opening Tableau Desktop.  
+
+
+## <a id="server"/> Kerberos RunAs and Delegation on Server
+When a user logs into Tableau desktop using Integrated authentication, there are the two publish options available for Kerberos authentication, namely, "Server run-as" and "Viewer Credentials/Kerberos Delegation".
```

Comment:
Logging in using integrated auth no longer required. Where did you see that it was required?

Reply:
I did not know we fixed that. I will test without that to check it works fine. Thanks